packages/rocketchat-ui/client/views/app/directory.html

```diff
@@ -1,87 +1,92 @@
 <template name="directory">
 	<section class="rc-directory">
 		{{# header sectionName="Directory" hideHelp=true}}
-		<div class="rc-directory-header">
-		<select class="rc-input__element rc-directory-selector js-typeSelector" name="type">
-				<option value="channels">{{_ 'Channels'}}</option>
-				<option value="users">{{_ 'Users' }}</option>
-			</select>
-			<div class="rc-input rc-input--small rc-directory-search">
-				<label class="rc-input__label">
-					<div class="rc-input__wrapper">
-						{{> icon icon="magnifier" block="rc-input__icon" }}
-						<input type="text" class="rc-input__element rc-input__element--small js-search" name="message-search" id="message-search" placeholder="{{_ 'Directory_Search_Placeholder'}}" autocomplete="off">
-					</div>
-				</label>
+			<div class="rc-directory-header">
+				<select class="rc-input__element rc-directory-selector js-typeSelector" name="type">
+					<option value="Channels">{{_ 'Channels'}}</option>
+					<option value="users">{{_ 'Users' }}</option>
+					<option value="Private_Groups">{{_ 'Private' }}</option>
+					{{#each roomType in customRoomTypes}}
+						<option value={{roomType}}> {{roomType}}</option>
+					{{/each}}
+				</select>
+				<div class="rc-input rc-input--small rc-directory-search">
+					<label class="rc-input__label">
+						<div class="rc-input__wrapper">
+							{{> icon icon="magnifier" block="rc-input__icon" }}
+							<input type="text" class="rc-input__element rc-input__element--small js-search"
+								   name="message-search" id="message-search"
+								   placeholder="{{_ 'Directory_Search_Placeholder'}}" autocomplete="off">
+						</div>
+					</label>
+				</div>
+				<button class="rc-button rc-button--small rc-button--primary rc-directory-plus">{{> icon icon="plus" }}</button>
 			</div>
-			<button class="rc-button rc-button--small rc-button--primary rc-directory-plus">{{> icon icon="plus" }}</button>
-		</div>
 		{{/header}}
 		<div class="rc-directory-content">
-			{{#if $eq searchType 'channels'}}
+			{{#if $eq searchType 'users'}}
```

Comment:
From a usability perspective, I'd wish for a note on top of the list, that - if rooms are secret - secret rooms will not be listed here.
Else, particularly after the first migration after which all rooms are secret, user's would glance at an empty list and not understand it.
Proposal:

```
{{#if secretRoomsExist}}
<div class=something-like-announcement>{{_ secret_rooms_not_listed}}<div>
{{/if}}
```

Reply:
WIP ;)